installer/PowerToysSetup/Core.wxs

```diff
@@ -47,33 +47,8 @@
     </DirectoryRef>
 
     <?if $(var.PerUser) = "true" ?>
-    <DirectoryRef Id="PersonalFolder">
-      <Directory Id="WindowsPowerShellFolder" Name="PowerShell">
-        <Directory Id="PowerShellModulesFolder" Name="Modules">
-          <Directory Id="PowerToysDscFolder" Name="Microsoft.PowerToys.Configure">
-            <Directory Id="PowerToysDscVerFolder" Name="$(var.Version)">
-              <Component Id="PowerToysDSC" Win64="yes" Guid="4A033E3B-6590-43FD-8FBD-27F9DF557F7F">
-                <RegistryValue Root="HKCU"
-                         Key="Software\[Manufacturer]\[ProductName]"
-                         Name="DSCInstalled"
-                         Type="integer"
-                         Value="1"
-                         KeyPath="yes"/>
-                <!-- Don't fail installation because of DSC. Files are marked as not vital. -->
-                <File Vital="no" Source="$(var.RepoDir)\src\dsc\Microsoft.PowerToys.Configure\Generated\Microsoft.PowerToys.Configure\$(var.Version)\Microsoft.PowerToys.Configure.psd1" Id="PTConf.psd1" />
-                <File Vital="no" Source="$(var.RepoDir)\src\dsc\Microsoft.PowerToys.Configure\Generated\Microsoft.PowerToys.Configure\$(var.Version)\Microsoft.PowerToys.Configure.psm1" Id="PTConf.psm1" />
-                <RemoveFolder Id="RemoveThisFolder" On="uninstall" />
-                <RemoveFolder Id="RemovePowerToysDscVerFolder" Directory="PowerToysDscVerFolder" On="uninstall" />
-                <RemoveFolder Id="RemovePowerToysDscFolder" Directory="PowerToysDscFolder" On="uninstall" />
-                <RemoveFolder Id="RemovePowerShellModulesFolder" Directory="PowerShellModulesFolder" On="uninstall" />
-                <RemoveFolder Id="RemoveWindowsPowerShellFolder" Directory="WindowsPowerShellFolder" On="uninstall" />
-              </Component>
-            </Directory>
-          </Directory>
-        </Directory>
-      </Directory>
-    </DirectoryRef>
-  <?else?>
+    <!--Nothing to do here; installation handled via custom action elsewhere -->
```

Comment:
Comment should say that it's the DSC module files that are handled elsewhere for user installation. Comment is a bit too vague right now. You can even name the custom actions here.